### ISSUE TITLE
Somewhat funded (question about the capacity)

### ISSUE THREAD

**Issue by techtonik** [jayfk/fundingoss.com#13]
@nayafia, the current choices for funding status are "Funded", "Self-Funded", "No funded" and "Unknown", but many projects are `partially` funded meaning that only few (or even one) developer works on the project (maybe part time), or `occasionally` funded meaning that somebody is occasionally does the work for bounty (provided that maintainers are still there to review and merge the requests).

**Comment by jayfk** [comment]
Yeah, that's why I have opened https://github.com/jayfk/fundingoss.com/issues/1

**Comment by techtonik** [comment]
Nice. I usually search for issues. =) So, keep this open for a discussion space?

**Comment by heyakyra** [comment]
maybe just merge the bugs

**Comment by techtonik** [comment]
"Somewhat funded" needs deeper investigation. Means "funded, but things could be better". To get to more theories why open source projects are underfunded and how to fix it, we need more data. And one of the metrics that I think would be useful is called **capacity**, which I see as two things:
1. how many people are needed to maintain the project (including bus factor)
2. how much money these people need to maintain their lives (including occasional contributions)

I am not able to define exact formula for project **capacity** right now (in terms of money, people or vespene gas), but some answers may at least help with starting:
- how many people are contributing to the project
- how many contributors have their basic (minimal) needs covered
- how many contributors are satisfied with their income, time and life balance
- who of those does major work (including reviews, testing, releasing)
- is there a lot of technical debt, is there an understanding of that is it
- who is dealing with that
- is there a lot of competence debt
- why people with the knowledge abandoned the project
- how many people abandoned project because of money/life support problem
- how many of those were able to reduce the technical debt and be useful to the project

So, how do you think - is it the idea of capacity definition is something that could be that useful for this research? Can it be improved?
